web/src/left_sidebar_navigation_area.js

```diff
@@ -119,4 +146,15 @@ function do_new_messages_animation($li) {
 
 export function initialize() {
     update_scheduled_messages_row();
+    $("body").on("mouseenter touchstart", "#left_sidebar_navigation_list li", () => {
+        if ($("#left_sidebar_navigation_area").hasClass("collapsed")) {
```

Comment:
Could this be solved in CSS? I'm not a fan of calling up `.css()` in general, and since all of this logic is about checking classes and selectors, it seems like a CSS-only solution should be possible. Failing that, I'd prefer toggling a class that takes these z-index values.

Reply:
I tried a CSS-only solution by making the z-index of the left sidebar greater than the z-index of the top bar in commit c5ab8b7f0467f90fe3623f8eeeb9e87a27c69216. Can you please confirm if this is the right approach or not?